sda-commons-server-morphia/src/test/java/org/sdase/commons/server/morphia/MorphiaBundleTracingIT.java

```diff
@@ -54,8 +54,8 @@ public void shouldHaveInstrumentation() {
             tracer.finishedSpans().stream()
                 .map(s -> s.tags().keySet())
                 .flatMap(Set::stream)
-                .noneMatch(Tags.DB_STATEMENT::equals))
-        .isTrue();
+                .filter(Tags.DB_STATEMENT.getKey()::equals))
+        .isNotEmpty();
```

Comment:
Before your change the test asserted that none matched the filter expression.
Now it assert that we have at least one match.
Just to confirm: The test was wrong before?
(I did not follow your discussion in the previous PR)

Reply:
As @JoergSiebahn stated:
> But that is the wrong assertion because we do not remove the statement from the trace but replace it with a sanitized version.

Seems to be ok then! 👍

Reply:
The test was wrong before, even though it still asserted correctly when removing the tracing support 😆